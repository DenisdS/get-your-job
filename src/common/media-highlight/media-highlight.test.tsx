import React from 'react'
import { mount } from 'enzyme'

import { MediaHighlightComponent } from './'

const createComponent = ({}) => {
  const wrapper = mount(<MediaHighlightComponent  />)
  const unmount = () => wrapper.unmount()

  return [wrapper, unmount]
}

describe('Header Component', () => {

  describe('Snapshot', () => {
    it('Should match snapshot', () => {
      const [wrapper, unmount] = createComponent({})

      expect(wrapper).toMatchSnapshot()
      unmount()
    })
  })
})
