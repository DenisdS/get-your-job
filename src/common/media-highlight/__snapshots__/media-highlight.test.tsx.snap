// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Snapshot Should match snapshot 1`] = `
ReactWrapper {
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "getWrappingComponentRenderer": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
    "wrapInvoke": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": Array [
      Object {
        "instance": <article>
          <h2>
            PALAVRA DO CEO
          </h2>
          <h3>
            Carlos Curioni
          </h3>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
          </p>
        </article>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2>
              PALAVRA DO CEO
            </h2>,
            <h3>
              Carlos Curioni
            </h3>,
            <p>
              Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
            </p>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": <h2>
              PALAVRA DO CEO
            </h2>,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "PALAVRA DO CEO",
            },
            "ref": null,
            "rendered": Array [
              "PALAVRA DO CEO",
            ],
            "type": "h2",
          },
          Object {
            "instance": <h3>
              Carlos Curioni
            </h3>,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Carlos Curioni",
            },
            "ref": null,
            "rendered": Array [
              "Carlos Curioni",
            ],
            "type": "h3",
          },
          Object {
            "instance": <p>
              Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
            </p>,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
            },
            "ref": null,
            "rendered": Array [
              "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
            ],
            "type": "p",
          },
        ],
        "type": "article",
      },
      Object {
        "instance": <h2 />,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": Array [
          undefined,
        ],
        "type": "h2",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": Array [
        Object {
          "instance": <article>
            <h2>
              PALAVRA DO CEO
            </h2>
            <h3>
              Carlos Curioni
            </h3>
            <p>
              Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
            </p>
          </article>,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                PALAVRA DO CEO
              </h2>,
              <h3>
                Carlos Curioni
              </h3>,
              <p>
                Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
              </p>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": <h2>
                PALAVRA DO CEO
              </h2>,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "PALAVRA DO CEO",
              },
              "ref": null,
              "rendered": Array [
                "PALAVRA DO CEO",
              ],
              "type": "h2",
            },
            Object {
              "instance": <h3>
                Carlos Curioni
              </h3>,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Carlos Curioni",
              },
              "ref": null,
              "rendered": Array [
                "Carlos Curioni",
              ],
              "type": "h3",
            },
            Object {
              "instance": <p>
                Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
              </p>,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
              },
              "ref": null,
              "rendered": Array [
                "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
              ],
              "type": "p",
            },
          ],
          "type": "article",
        },
        Object {
          "instance": <h2 />,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": Array [
            undefined,
          ],
          "type": "h2",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__linkedRoots__): Array [],
  Symbol(enzyme.__unrendered__): <MediaHighlightComponent />,
  Symbol(enzyme.__updatedBy__): null,
}
`;
